server/src/main/java/org/candlepin/pinsetter/tasks/HypervisorUpdateJob.java

```diff
@@ -167,23 +164,6 @@ public void toExecute(final JobExecutionContext context) throws JobExecutionExce
             final HypervisorUpdateAction.Result updateResult = hypervisorUpdateAction.update(
                 owner, hypervisors, create, principal, jobReporterId);
             final HypervisorUpdateResultDTO result = updateResult.getResult();
-            final VirtConsumerMap hypervisorKnownConsumersMap = updateResult.getKnownConsumers();
-
-            final List<Consumer> created = new ArrayList<>();
-            final List<Consumer> updated = new ArrayList<>();
-            for (Consumer consumer : hypervisorKnownConsumersMap.getConsumers()) {
-                final HypervisorConsumerDTO translated = this.translator.translate(
-                    consumer, HypervisorConsumerDTO.class);
-                if (result.wasCreated(translated)) {
-                    created.add(consumer);
-                }
-                else {
-                    updated.add(consumer);
-                }
-            }
-
-            doInBulk(created, consumers -> consumerCurator.saveAll(consumers, false, false));
```

Comment:
We should remove doInBulk as it's unused. There are also some unused dependencies.